data-prepper-plugins/cloudwatch-logs/src/integrationTest/java/org/opensearch/dataprepper/plugins/sink/cloudwatch_logs/CloudWatchLogsIT.java

```diff
@@ -542,6 +543,36 @@ void testWithBadCredentials_AllEventsToDLQ() {
 
     }
 
+    @Test
+    public void testToVerifyLackOfCredentialsResultInFailure() throws Exception {
+        long startTime = Instant.now().toEpochMilli();
+        when(thresholdConfig.getBatchSize()).thenReturn(1);
+        when(thresholdConfig.getMaxEventSizeBytes()).thenReturn(1000L);
+        when(thresholdConfig.getMaxRequestSizeBytes()).thenReturn(1000L);
+        AwsCredentialsProvider provider = mock(AwsCredentialsProvider.class);
+        when(awsCredentialsSupplier.getProvider(any())).thenReturn(provider);
+
+        sink = createObjectUnderTest();
+        Collection<Record<Event>> records = getRecordList(NUM_RECORDS);
+        sink.doOutput(records);
+
+        assertThrows( org.awaitility.core.ConditionTimeoutException.class, () ->  await().atMost(Duration.ofSeconds(30))
```

Comment:
As I understand this, Awaitility will keep looking for this assertion for 30 seconds. And in the end, you don't want it to succeed.

In this case, it might be clearer to just use a `sleep`. That is all you are trying to do - determine the end state after 30 seconds.

Also, can we reduce this time? Thirty seconds is going to be a long time in our tests.